okhttp-coroutines/src/main/kotlin/okhttp3/coroutines/ExecuteAsync.kt

```diff
@@ -45,8 +43,8 @@ suspend fun Call.executeAsync(): Response =
           call: Call,
           response: Response,
         ) {
-          continuation.resume(response) {
-            response.closeQuietly()
+          continuation.resume(response) { _, value, _ ->
+            value.closeQuietly()
```

Comment:
This one’s not deprecated

https://github.com/Kotlin/kotlinx.coroutines/blob/f4f519b36734238ec686dfaec1e174086691781e/kotlinx-coroutines-core/common/src/CancellableContinuation.kt#L304